roles/ovb/bmc/files/bmc_listener.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+# Copyright 2015 Red Hat, Inc.
+# Copyright 2015 Lenovo
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Virtual BMC for controlling OpenStack instances, based on fakebmc from
+# python-pyghmi
+
+# Sample ipmitool commands:
+# ipmitool -I lanplus -U admin -P password -H 127.0.0.1 power on
+# ipmitool -I lanplus -U admin -P password -H 127.0.0.1 power status
+# ipmitool -I lanplus -U admin -P password -H 127.0.0.1 chassis bootdev pxe|disk
+# ipmitool -I lanplus -U admin -P password -H 127.0.0.1 mc reset cold
+
```

Comment:
Could you please add a short description here with a link to the source?
Are there any modifications you did for the project, or is it simply a copy-paste?

```python
"""
This script is copied from <url>
With the following changes: ...

Ansible will launch it as a deamon for each of the overcloud nodes
"""
```